src/sas/qtgui/Perspectives/Invariant/UnitTesting/InvariantPerspectiveTest.py

```diff
@@ -121,46 +93,59 @@ def testDefaults(self):
         self.assertIsInstance(self.widget.txtNptsHighQ.validator(), QtGui.QIntValidator)
         self.assertIsInstance(self.widget.txtPowerLowQ.validator(), GuiUtils.DoubleValidator)
         self.assertIsInstance(self.widget.txtPowerHighQ.validator(), GuiUtils.DoubleValidator)
-
         self.assertIsInstance(self.widget.txtBackgd.validator(), GuiUtils.DoubleValidator)
         self.assertIsInstance(self.widget.txtContrast.validator(), GuiUtils.DoubleValidator)
         self.assertIsInstance(self.widget.txtScale.validator(), GuiUtils.DoubleValidator)
         self.assertIsInstance(self.widget.txtPorodCst.validator(), GuiUtils.DoubleValidator)
 
-        # Test autoexclusivity of radiobuttons
-        # Low Q
-        self.assertFalse(self.widget.rbGuinier.autoExclusive())
-        self.assertFalse(self.widget.rbPowerLawLowQ.autoExclusive())
-        self.assertTrue(self.widget.rbFixLowQ.autoExclusive())
-        self.assertTrue(self.widget.rbFitLowQ.autoExclusive())
-        # High Q
-        self.assertTrue(self.widget.rbFixHighQ.autoExclusive())
-        self.assertTrue(self.widget.rbFitHighQ.autoExclusive())
+    def checkControlDefaults(self):
+        # All values in this list should assert to False
+        false_list = [
+            self.widget._allow_close, self.widget.allowBatch(),
+            # disabled buttons
+            self.widget.cmdStatus.isEnabled(), self.widget.cmdCalculate.isEnabled(),
+            # read only text boxes
+            self.widget.txtBackgd.isReadOnly(), self.widget.txtScale.isReadOnly(), self.widget.txtContrast.isReadOnly(),
+            self.widget.txtPorodCst.isReadOnly(), self.widget.txtNptsLowQ.isReadOnly(),
+            self.widget.txtNptsHighQ.isReadOnly(), self.widget.txtName.isEnabled(),
+            # unchecked check boxes
+            self.widget.chkLowQ.isChecked(), self.widget.chkHighQ.isChecked(),
+            # radio buttons exclusivity
+            self.widget.rbGuinier.autoExclusive(), self.widget.rbPowerLawLowQ.autoExclusive()
+            ]
+        # All values in this list should assert to True
+        true_list = [
+            # Enable buttons
+            self.widget.txtExtrapolQMax.isEnabled(), self.widget.txtExtrapolQMin.isEnabled(),
+            # enabled text boxes
+            self.widget.txtVolFract.isReadOnly(), self.widget.txtVolFractErr.isReadOnly(),
+            self.widget.txtSpecSurf.isReadOnly(), self.widget.txtSpecSurfErr.isReadOnly(),
+            self.widget.txtInvariantTot.isReadOnly(), self.widget.txtInvariantTotErr.isReadOnly(),
+            # radio buttons exclusivity
+            self.widget.rbFixLowQ.autoExclusive(), self.widget.rbFitLowQ.autoExclusive(),
+            self.widget.rbFixHighQ.autoExclusive(), self.widget.rbFitHighQ.autoExclusive()
+            ]
+        self.assertTrue(all(v is False for v in false_list))
+        self.assertTrue(all(v is True for v in true_list))
```

Comment:
Those tests are sort of silly, the default state is tested with the GUI testing with Squish.